Hyper/Shader.cs

```diff
@@ -1,142 +1,83 @@
-﻿using System;
-using System.IO;
-using System.Text;
-using System.Collections.Generic;
-using OpenTK.Graphics.OpenGL4;
+﻿using OpenTK.Graphics.OpenGL4;
 using OpenTK.Mathematics;
 
 namespace Hyper
 {
-    // A simple class meant to help create shaders.
     public class Shader
     {
         public readonly int Handle;
 
         private readonly Dictionary<string, int> _uniformLocations;
 
-        // This is how you create a simple shader.
-        // Shaders are written in GLSL, which is a language very similar to C in its semantics.
-        // The GLSL source is compiled *at runtime*, so it can optimize itself for the graphics card it's currently being used on.
-        // A commented example of GLSL can be found in shader.vert.
         public Shader(string vertPath, string fragPath)
         {
-            // There are several different types of shaders, but the only two you need for basic rendering are the vertex and fragment shaders.
-            // The vertex shader is responsible for moving around vertices, and uploading that data to the fragment shader.
-            //   The vertex shader won't be too important here, but they'll be more important later.
-            // The fragment shader is responsible for then converting the vertices to "fragments", which represent all the data OpenGL needs to draw a pixel.
-            //   The fragment shader is what we'll be using the most here.
-
-            // Load vertex shader and compile
             var shaderSource = File.ReadAllText(vertPath);
-
-            // GL.CreateShader will create an empty shader (obviously). The ShaderType enum denotes which type of shader will be created.
             var vertexShader = GL.CreateShader(ShaderType.VertexShader);
-
-            // Now, bind the GLSL source code
             GL.ShaderSource(vertexShader, shaderSource);
-
-            // And then compile
             CompileShader(vertexShader);
 
-            // We do the same for the fragment shader.
             shaderSource = File.ReadAllText(fragPath);
```

Comment:
Repetition. Should probably have a function for setting up a shader since we will have more than 2 in the future